lib/client-assets.php

```diff
@@ -159,13 +159,19 @@ function gutenberg_register_scripts_and_styles() {
 		'before'
 	);
 
-	// The editor code itself.
 	wp_register_script(
 		'wp-editor',
 		gutenberg_url( 'editor/build/index.js' ),
 		array( 'postbox', 'jquery', 'wp-api', 'wp-data', 'wp-date', 'wp-i18n', 'wp-blocks', 'wp-element', 'wp-components', 'wp-utils', 'word-count', 'editor', 'heartbeat' ),
-		filemtime( gutenberg_dir_path() . 'editor/build/index.js' ),
-		true // enqueue in the footer.
+		filemtime( gutenberg_dir_path() . 'editor/build/index.js' )
+	);
+
+	wp_register_script(
+		'wp-edit-post',
+		gutenberg_url( 'edit-post/build/index.js' ),
+		array( 'wp-element', 'wp-components', 'wp-editor', 'wp-i18n', 'wp-date', 'wp-utils' ),
```

Comment:
`edit-post/index.js` has a reference to jQuery and heartbeat, so this should be made an explicit dependency.

Heartbeat is no longer used as a dependency in `wp-editor`.